client/state/themes/selectors/get-theme.js

```diff
@@ -10,8 +11,21 @@ import 'calypso/state/themes/init';
  * @returns {?Object}         Theme object
  */
 export const getTheme = createSelector(
-	( state, siteId, themeId ) => {
-		const manager = state.themes.queries[ siteId ];
+	( state, sourceAndSiteId, themeId ) => {
```

Comment:
`SourceAndSiteId` might not be the best name, does this variable hold the source or the siteId?

Reply:
It can include both. In some cases, we pass the site id while in others we pass the source...

Reply:
Don't you consider that a smell that maybe there is something wrong with the implementation?